### PR TITLE
Claim storage initailized

### PR DESCRIPTION
**Pull request by Akado2009**

### CODE REVIEW

File: docs/examples/claim-storage.md [hunk 1]
```diff
@@ -0,0 +1,394 @@
...
+# Claim Storage
+
+## Introduction
+
+This smart contracts example created by Gear represents a Simple Claim storage.
+
+It is a simple storage operating with Claims. A claim is a basic strucure which states some fact about a subject. A claim can be issued by a subject, or any other issuer on behalf of the subject (e.g. a university might issue the claim that a subject has successfully graduted).
+Whenever a claim is issued it MUST be signed by an issuer (or a subject in case he issues it on behalf of himself).
+
+Every other contract user can verify any claim with his public key and a signature or check the claim data using one of the claim set's hash (more detail bellow).
```
Comment (shamilsan) [inline docs/examples/claim-storage.md:15]:
```suggestion
Every other contract user can verify any claim with his public key and a signature or check the claim data using one of the claim set's hashes (more detail below).
```

File: docs/examples/claim-storage.md [hunk 2]
```diff
@@ -0,0 +1,394 @@
...
+    pub issuer_signature: Signature,
+    /// Subject's public key.
+    pub subject: PublicKey,
+    /// Map of verifiers PublicKey -> Signature
+    pub verifiers: BTreeMap<PublicKey, Signature>,
+    /// Internal data of the claim
+    pub data: ClaimData,
+}
+```
+Internally claim contains an issuance date, a validity status and a set of hashed data. The set is used the following way. Say, one wants to store his address: Russia, Moscow, Nikolskaya street, 3, 15. This is okay if the users wants to be able to verify the whole address. But if a user wants to only verify that he lives in Moscow? Should there be another claim created? The answer is no. A user simply breaks his address and hashed every value seprately and than passes it as a set. The check procedure is done against the whole set, so one can now verify he lives in Moscow without disclosing the full address.
```
Comment (shamilsan) [inline docs/examples/claim-storage.md:55]:
```suggestion
Internally claim contains an issuance date, a validity status, and a set of hashed data. The set is used the following way. Say, one wants to store his address: Russia, Moscow, Nikolskaya street, 3, 15. This is okay if the user wants to be able to verify the full address. But if a user wants only to verify that he lives in Moscow? Should there be another claim created? The answer is no. A user simply breaks his address and hashed every value separately and then passes it as a set. The check procedure is done against the whole set, so one can now verify he lives in Moscow without disclosing the full address.
```

File: docs/examples/claim-storage.md [hunk 3]
```diff
@@ -0,0 +1,394 @@
...
+                status,
+            },
+            0,
+        )
+        .expect("IDENTITY: Error during replying with IdentityEvent::CheckedClaim");
+    }
+```
+
+### Testing
+Though the contract is not that complicated itself, we strongly suggest you check the tests. In there provided tests we show a user actually interacts with the contract (e.g. keypairs/signature and hasing).
```
Comment (shamilsan) [inline docs/examples/claim-storage.md:387]:
```suggestion
Though the contract is not that complicated itself, we strongly suggest you check the tests. In there provided tests we show a user actually interacts with the contract (e.g. keypairs/signature and hashing).
```

**Review by shamilsan — COMMENTED** [review]
I'd recommend using a spell checker. And please replace Moscow etc with more neutral things.

**Review by shamilsan — APPROVED** [review]
Nobody wants to review it 😄

**Comment by xrdavies** [comment]
> Nobody wants to review it 😄

Should we merge it or not?

**Comment by shamilsan** [comment]
> Should we merge it or not?

@AndrePanin what do you think?

**Comment by xrdavies** [comment]
it seems no one want review it, so let's close it.

**Pull request closed without merging**